gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -377,3 +377,22 @@ def add_arguments(self, parser):
         help=('The full path of the resolved headers. The file will not be'
               'generated if unspecified. Otherwise, please provide a local '
               'path if run locally, or a cloud path if run on Dataflow.'))
+    parser.add_argument(
```

Comment:
why was this added? :)

Reply:
Sorry, I somehow messed up while syncing.